modules/rosa-troubleshooting-expired-token.adoc

```diff
@@ -5,7 +5,7 @@
 
 
 [id="rosa-troubleshooting-expired-token_{context}"]
-= Troubleshooting an expired offline access token
+= Expired offline access token
```

Comment:
Procedure titles need to start with a [gerund](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings).